components/availability/src/Availability.svelte

```diff
@@ -1070,7 +1069,7 @@
             display: none;
             position: relative;
             z-index: 2;
-            text-align: right;
+            float: right;
```

Comment:
Side-effect: had been using text-align for the floating "x of y" availability button, but realized that prevented me from clicking the timeslot anywhere on the same vertical level of it. cc @heathergh